Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -2,18 +2,24 @@
 title: CephBlockPool CRD
 ---
 
-Rook allows creation and customization of storage pools through the custom resource definitions (CRDs). The following settings are available for pools.
+Rook enables creation and customization of Ceph block storage (RBD) pools through custom
+resource definitions (CRDs). Settings and options are detailed below.
 
 ## Examples
 
-### Replicated
+### Replicated RBD Pool
 
-For optimal performance, while also adding redundancy, this sample will configure Ceph to make three full copies of the data on multiple nodes.
+For optimal performance, while also adding redundancy, this example configures
+a Ceph RBD pool with three full copies of user data, each on three different nodes.
 
 !!! note
-    This sample requires *at least 1 OSD per node*, with each OSD located on *3 different nodes*.
+    This sample requires at least one OSD on each of at least three nodes.
```

Comment:
IMO, the original reads more clearly than the rewrite here. I can imagine someone who doesn't have English as a native language getting confused here.

Reply:
Upstream Ceph follows the conventional English practice of spelling out small numbers.  This aligns with that.